app/src/main/java/com/cornellappdev/android/eatery/network/NetworkUtilities.java

```diff
@@ -63,14 +72,30 @@ public static String getJSON() {
       @Override
       public void onResponse(@NotNull Response<AllEateriesQuery.Data> response) {
         eateries = response.data().eateries();
-        EATERIES_LOADED = true;
+        ArrayList<EateryBaseModel> eateryList = JsonUtilities.parseEateries(eateries,activity);
+        Collections.sort(eateryList);
+        presenter.setEateryList(eateryList);
+
+        // Runs on MainActivity's UI Thread
+        activity.runOnUiThread(new Runnable() {
+          @Override
+          public void run() {
+            try {
+              ((MainActivity)activity).getSupportFragmentManager()
+                  .beginTransaction()
+                  .replace(R.id.frame_fragment_holder, new MainListFragment())
+                  .commit();
+            } catch (Exception e) {
+              Log.e(TAG, e.getMessage());
```

Comment:
I'm not a huge fan of Log. Instead, this should be `e.printStackTrace()`

Reply:
Done